### PR TITLE
Initial modification to Tradheli guide

### PR DESCRIPTION
**Pull request by bnsgeyer**
Here is the start of modification to the Tradheli guide.   This is draft and still needs a lot of work.

### CODE REVIEW

File: TUNING_GUIDE_Heli.md [hunk 1]
```diff
@@ -36,16 +36,13 @@ You will be able to tune multiple vehicles (think production line) using this me
 
 While [choosing an Autopilot](https://ardupilot.org/copter/docs/common-autopilots.html) and [other hardware](https://ardupilot.org/copter/docs/common-optional-hardware.html) components [avoid these components](https://discuss.ardupilot.org/t/hardware-to-avoid-when-building-your-first-multirotor/114014/1)
 
-Use tools like [ecalc for multirotor](https://www.ecalc.ch/index.htm) to find a suitable set of components to meet your needs.
 
-![eCalc example screenshot](images/blog/eCalc_(en).PNG)
```
Comment (amilcarlucas) [inline TUNING_GUIDE_Heli.md:41]: There is ecalc Heli

so just replace multirotor with heli

**Review by amilcarlucas — CHANGES_REQUESTED** [review]
Nice, thanks for helping out with this

**Comment by amilcarlucas** [comment]
https://github.com/ArduPilot/MethodicConfigurator/releases/tag/latest contains a Heli OMP_M4 template that @allister is working on. Should we use that instead of the Trex500 ?

**Comment by Allister779** [comment]
Let me know if you want any changes to the OMP_M4 files.

**Comment by amilcarlucas** [comment]
I did the changes locally (I could not push to your repo) and merged this one. Please create a new one to continue to work on this

**Pull request closed without merging**